### ISSUE TITLE
TS2488 When use object in v-for

### ISSUE THREAD

**Issue by cawa-93** [johnsoncodehk/vue-tsc#3]
I have [component](https://github.com/cawa-93/vite-electron-builder/blob/main/packages/renderer/src/components/About.vue):
```vue
<template>
...
      <li
        v-for="(version, lib) in versions"
        :key="lib"
      >
        <strong>{{ lib }}</strong>: v{{ version }}
      </li>
...
</template>

<script lang="ts">
export default defineComponent({
  setup() {
    const {versions} = useElectron()
    // It makes no sense to make "versions" reactive
    return {versions} // Version is ftat object {node: '14', chrome: '86' ... }
  },
})
</script>
``` 
In Runtime it works as expected. But checking the types [falls with a error](https://github.com/cawa-93/vite-electron-builder/pull/162/checks?check_run_id=2122051574):
```
Error: src/components/About.vue(8,34): error TS2488: Type 'Readonly<Record<string, string>>' must have a '[Symbol.iterator]()' method that returns an iterator.
```

**Comment by johnsoncodehk** [comment]
This is expected, you can add ` & { [Symbol.iterator](): IterableIterator<FOR_TYPE> }` to your type for fix.

**Comment by cawa-93** [comment]
OK. This really fixed the problem. I wonder what caused this behavior. It doesn't seem intuitive.

**Comment by johnsoncodehk** [comment]
For example:

```ts
declare let foo: Record<string, string>;
for (const item of foo) { } // TS2488 

declare let bar: Record<string, string> & { [Symbol.iterator](): IterableIterator<string> };
for (const item of bar) { } // works
```

**Comment by johnsoncodehk** [comment]
I found that `for...in` can better to handle it, and it was fixed in `vue-tsc` 0.0.14. Sorry for make the confusion.
